tests/ApiTest.php

```diff
@@ -320,4 +320,40 @@ public function testMaxBytesOverride(): void
         ]);
         $this->assertSame(123, Repository::$maxBytes);
     }
+
+    /**
+     * @covers ::getTargetFromUrl
```

Comment:
```suggestion
     * @covers ::prepareMetadata
```

Since this is the public function?

Reply:
True, but this test is _really_ meant to target `getTargetFromUrl()`. I'd rather leave it as-is for accuracy.